optaweb-tsp-ui/.gitignore

```diff
@@ -1,5 +1,6 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
-
+### Maven Frontend Plugin ###
+node/
```

Comment:
@danielefiungo Can we put the node output under the target directory?

Reply:
Yes, I'll add the required config